python/examples/structure_map.py

```diff
@@ -83,3 +100,9 @@
         "structure": [{"keepOrientation": True, "playbackDelay": 100}]
     },
 )
+cs
+
+# %%
+#
+# Save as a file that can be viewed at chemiscope.org
+cs.save("trajectory-pca.json.gz")
```

Comment:
```suggestion
```

Reply:
I think it is nice to also show how to export the file, which requires defining a cs object and then displaying it.
This also means that when a user runs the examples from the CLI they'll get some output otherwise they'll just get an error.

Reply:
Yes, but we don't have to do this in all examples, it is already showed in `base.py`.

Also, the examples can now be dowloaded as jupyter notebooks, which is what I expect most users will be doing. And finally, when running on the CLI, users should be using `chemiscope.write_input` directly and not go through jupyter just for fun 😃

Reply:
I think people can legitimately want to run these example from python. I agree we could use write_input but that would mean copying and pasting all the arguments to show() that seems silly to me. maybe we could write_input and then have a show_input() call instead?

Reply:
I think that examples should be focused on one thing, and not everything at the same time.

> maybe we could write_input and then have a show_input() call instead?

I don't understand what you mean here. What would show_input do?

Reply:
The examples are there to demonstrate chemiscope functionalities. The fact that they work both to output a static .json.gz and an interactive viewer seems to me a good thing. 

`show_input` loads an input file from disk and shows it. so we would first write_input the .json.gz and then load it to view it in a notebook.